tests/Feature/Http/Controllers/ScoreLeaderboardControllerTest.php

```
@@ -0,0 +1,214 @@
+        $computedAt = now()->subDays(30);
+        LeaderboardEntry::create([
+            'login' => 'jane',
+            'board' => 'contributor',
+            'window' => 'rolling12',
+            'score' => 10.0,
+            'rank' => 1,
+            'computed_at' => $computedAt,
+        ]);
+
+        $capturedTo = null;
+        $windowDays = (int) config('leaderboard.recency.window_days', 365);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$windowDays'. (undefined)

(UnusedLocalVariable)